__tests__/app.test.js

```diff
@@ -34,7 +33,42 @@ describe('GET /api', ()=>{
         .expect(200)
         .then(({body})=>{
         expect(body.endPoints).toEqual(endpoints)
-           
         })
     });
-})
\ No newline at end of file
+})
+
+describe('GET /api/articles/4', ()=>{
+    test('should return a status 200 if the data is found and return a single article of the correct id',() =>{
+        return request(app)
+        .get('/api/articles/4')
+        .expect(200)
+        .then(({body})=>{
+            console.log(body)
+        expect(body.article.article_id).toBe(4)
```

Comment:
This is spot on, but we should also have other expect statements in here for the other keys that will come back. Think about whether we need to be explicit with those, or could we just expect them to return back a certain data type for each value on the key?